components/next_steps_view/steps/complete_profile_step/complete_profile_step.tsx

```diff
@@ -0,0 +1,173 @@
+    }
+
+    render() {
+        const {currentUser} = this.props;
+
+        // Make sure picture has been set
+        const pictureSrc = currentUser.last_picture_update ? Utils.imageURLForUser(currentUser.id, currentUser.last_picture_update) : undefined;
+        const defaultSrc = Utils.defaultImageURLForUser(currentUser.id);
+
+        return (
+            <div className='NextStepsView__stepWrapper'>
+                <div className='CompleteProfileStep'>
+                    <div className='CompleteProfileStep__profilePicture'>
+                        <h3>
+                            <FormattedMessage
+                                id='next_steps_view.complete_profile_step.addAPhoto'
+                                defaultMessage='Add a photo'
+                            />
+                        </h3>
+                        <PictureSelector
+                            onSelect={this.onSelectPicture}
+                            onRemove={this.onRemovePicture}
+                            src={pictureSrc}
+                            defaultSrc={defaultSrc}
+                        />
+                    </div>
+                    <div className='CompleteProfileStep__fullName'>
+                        <h3>
+                            <FormattedMessage
+                                id='next_steps_view.complete_profile_step.enterYourName'
+                                defaultMessage='Enter your name'
+                            />
+                        </h3>
+                        <Input
+                            name='fullName'
+                            type='text'
+                            value={this.state.fullName}
+                            onChange={this.handleInputChange}
+                            placeholder={Utils.localizeMessage('next_steps_view.complete_profile_step.yourFullName', 'Your full name')}
+                            error={this.state.fullNameError}
+                            info={Utils.localizeMessage('next_steps_view.complete_profile_step.nameWillBeDisplayed', 'Your name will be displayed with your messages')}
+                        />
+                    </div>
+                </div>
+                <span className='CompleteProfileStep__pictureError'>
+                    {this.state.profilePictureError && (
+                        <>
+                            <i className='icon icon-alert-outline'/>
+                            <FormattedMarkdownMessage
+                                id='next_steps_view.complete_profile_step.pictureError'
+                                defaultMessage='Photos must be in BMP, JPG or PNG format. Maximum file size is {max}.'
+                                values={{max: Utils.fileSizeToString(this.props.maxFileSize)}}
+                            />
+                        </>
+                    )}
+                </span>
+                <div className='NextStepsView__wizardButtons'>
+                    {/* <button
```

Comment:
Remove?

Reply:
This might actually come back, we removed it for UX reasons. @matthewbirtch thoughts on just blowing this out?

Reply:
If that's the case I'll approve for now.